transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringCompletionQueue.java

```diff
@@ -39,6 +39,9 @@
   private final int ringSize;
   private final long ringAddress;
   private final int ringFd;
+  
```

Comment:
Checkstyle seems to complain about trailing whitespace on this line.

Reply:
:+1: I hadn't been running these through checkstyle up to now but will do and make sure it passes